applications/osb-portal/src/components/workspace/drawer/WorkspaceResourceBrowser.tsx

```diff
@@ -27,6 +28,9 @@ const openFileResource = (resource: WorkspaceResource, refreshWorkspace: any) =>
   return workspaceResourceService.workspacesControllerWorkspaceResourceOpen(resource.id).then(() => {
     const iFrame: HTMLIFrameElement = document.getElementById("workspace-frame") as HTMLIFrameElement;
     iFrame.contentWindow.postMessage(fileName, '*');
+    console.log('resource', resource);
+    const domain = getBaseDomain();
+    iFrame.src = `//${resource.type.application.subdomain}.${domain}/hub/spawn/hub/spawn//${resource.workspaceId}${resource.type}`;
```

Comment:
why is hub/spawn doubled?

Reply:
It was a typo, I'll remove the extra one.